src/main/resources/templates/team.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/extras/spring-security">
+
+    <head th:replace="c_head :: head"></head>
+
+    <body>
+
+        <header th:replace="c_header :: header"></header>
+
+        <!--<header th:replace="c_nav_m :: nav_m"></header>-->
+
+        <!--Basic group information-->
+        <div class="shades">
+            <span th:text="'Name:' + ${team}"></span>
+            <span th:text="'Description:' + ${team.description}"></span>
+            <span th:text="'Difficulty:' + ${team.difficulty}"></span>
+            <span th:text="'Current Size:' + ${team.users.size()} + '/' + ${team.capacity}"></span>
+            <span th:text="'Created:' + ${team.dateCreated}"></span>
+            <span th:text="'Going On:' + ${team.dateGoing}"></span>
+
+            <!--List of memebers in the group. Only shown to logged in users-->
+            <div id="users" sec:authorize="isAuthenticated()">
+                <div th:each="user : ${team.users}">
+                    <span th:text="${user.username}"></span>
+                    <span th:text="${user.firstName} + ' ' + ${user.lastName}"></span>
+                    <span th:text="${user.bio}"></span>
+                </div>
+            </div>
+
+            <!--Option to join the group, only available to logged in users-->
+            <form sec:authorize="isAuthenticated()" th:if="(${team.users.size()} < ${team.capacity} and !${inGroup})" th:action="'/teams/' + ${team.id}" method="post">
```

Comment:
Typically to have good RESTful routing, I'd assume this would post to `/teams/id/users`.